tests/roles/vm_image.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Create volumes
+  hosts: localhost
+
+  vars:
+    kubevirt_image_volumes:
+      - name: volume1
+        namespace: default
+        size: 100Mi
+        source: "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
+
+      - name: volume2
+        namespace: default
+        size: 100Mi
+        source: blank
+
+      - name: volume3
+        namespace: default
+        size: 100Mi
+
+  roles:
+    - vm_image
```

Comment:
Maybe add also some cleanup playbook? So we don't waste with resources?

Reply:
The role tests are run only on Travis in a dedicated job and the dedicated VM gets immediately removed afterwards. If we ever get to some more elaborate role testing – sure, might be needed to add a removal step (and maybe stricter verification steps, more in line with what, say, `kubevirt_vm.yml` has). But for now adding object removal would only make the tests run a tiny bit longer without any gain.